src/main.ts

```diff
@@ -52,9 +52,10 @@ async function run() {
         issue.number
       );
     } else {
-      firstContribution = await isFirstPull(
+      firstContribution = await isFirstOpenedOrMergedPR(
         client,
         sender,
+        `${context.repo.owner}/${context.repo.repo}`,
```

Comment:
Why is this necessary? This seems to go against the (albeit questionable) change you have made in https://github.com/zephyrproject-rtos/action-first-interaction/commit/5d55935c19f4b6f45eaa113304c50487f53353fb#diff-7934bf411fea192ad8cd69e0a12911648a2842cb0f2409a8fb67b41b7069d757L128.

Reply:
Good call. 100% an oversight.